docs/topics/contributing-starting-preview-server.adoc

```diff
@@ -0,0 +1,34 @@
+
+= Starting Preview Server
+
+You can preview the Launchpad locally. To do that, perform the following steps:
```

Comment:
I think the Launchpad word is discouraged, and also this doesn't really preview the Launch app -- it previews the documentation.